packages/esm-patient-search-app/translations/en.json

```diff
@@ -5,12 +5,15 @@
   "clear": "Clear",
   "clearSearch": "Clear",
   "closeSearch": "Close Search Panel",
-  "countOfFiltersApplied": "filters applied",
+  "countOfFiltersApplied": "filters applied {{count}}",
```

Comment:
```suggestion
  "countOfFiltersApplied": "{{count}} filters applied",
```

Reply:
Remove the key value from the translation file and then add it again.
The pluralization feature from i18next hasn't kicked in the translation file